burn-tch/src/ops/base.rs

```diff
@@ -275,6 +275,11 @@ impl<E: tch::kind::Element + Copy + Default> TchOps<E> {
         )
     }
 
+    pub fn maximum<const D: usize>(lhs: TchTensor<E, D>, rhs: TchTensor<E, D>) -> TchTensor<E, D> {
```

Comment:
I suggest we use `max` function/method name to shorten `maximum` since it's so commonly used (python, pytorch, rust, etc).

Reply:
Also, for the function, I don't think you need to use lhs (left hand side) and rhs (right handside). These two params are used for binary operation (add, sub). max is a unary operation.

Reply:
this is the function I'm aiming to support: https://pytorch.org/docs/stable/generated/torch.maximum.html

Reply:
I can add the unary max as well though; only a bit blocked by not knowing how all of the types and traits fit together across crates